src/components/ui/navbar/index.tsx

```diff
@@ -7,8 +7,23 @@ const NavBar = () => {
     <Navbar>
       <Logo />
       <Links>
-        <LinksPage>GitHub</LinksPage>
-        <LinksPage>About</LinksPage>
+        <LinksPage
+          onClick={() =>
+            window.open('https://github.com/VictorYarosh/weather', '_blank')
+          }
+        >
```

Comment:
Why so complicated? 
use just <a href="...." >
This is a simple link, not a button.

Reply:
fix this